modules/nf-core/seurat/multiseq/tests/nextflow.config

```diff
@@ -0,0 +1,42 @@
+params{
+    //optional parameters for MULTIseq and parameter to start the visualisation module
+    assay = "HTO"
+    nfeatures = 2000
+    quantile = 0.7
+    autoThresh = false
+    maxiter = 5
+    qrange_from = 0.1
+    qrange_to = 0.9
+    qrange_by = 0.05
+    verbose = true
+    selection_method = "mean.var.plot"
+    normalization_method = "CLR"
+
+    // Parameters to generate plots
+    group_cells_feature_scatter = "MULTI_ID"
+    feature_scatter_feature_1 = "MS-11"
+    feature_scatter_feature_2 = "MS-12"
+    number_of_features_ridge_plot = 2
+    number_of_cols_ridge_plot =  2
+    group_cells_violin_plot = "MULTI_classification"
+    features_violin_plot = "nCount_RNA"
+    pt_size = 0.1
+    log = true
+    subset_idents = "Negative"
+    subset_invert = true
+    tsne_scale_data_verbose = false
+    run_pca_approx = false
+    run_tsne_dim_max = 2
+    run_tsne_perplexity = 100
+    check_duplicates_tsne = false
+    resolution = 0.6
+    singlet_identities_tsne = "MULTI_classification"
```

Comment:
Are these all optional? Does it work without this?

Reply:
For all the parameters before  `// Parameters to generate plots`, the tool has the values by default. 
After  `// Parameters to generate plots`, some names such as `feature_scatter_feature_1 = "MS-11"` depend on the data and, therefore, must be given, if the option produce_plots is true, should I include those parameters as inputs in the `main.nf`?